package.json

```diff
@@ -34,6 +34,7 @@
     "@types/jasmine": "~2.8.3",
     "@types/jasminewd2": "~2.0.2",
     "@types/node": "^12.11.1",
+    "@int/geotoolkit": "^3.0.56",
```

Comment:
I think you need to add "@int/impl": "^3.0.56", too before

Reply:
It was not used any where. When I tested, the local build passed and the application loaded without any console errors. Hence did not add.

But if there is any dependency or any case I may have not tested, then I will add it.

Your thoughts?

Reply:
yes, geotoolkit depends on it

Reply:
Fixed.